model/connect.js

```diff
@@ -0,0 +1,17 @@
+
+function connect(username, password) {
```

Comment:
Where are we getting this username/password? I would recommend environment variables. That will have to be implemented in a separate config.js class separate from the model.

Reply:
I don't want to put username/password combos onto public repository, this will be unique for each local mysql server unless we synchronize